docs/docs-content/clusters/public-cloud/aws/eks.md

```diff
@@ -1,172 +1,270 @@
 
-16. Review the settings summary and click on **Finish Configuration** to deploy the cluster. Be aware that provisioning an AWS EKS clusters can take several minutes.
+You can access your Kubernetes cluster by using the kubectl CLI. Refer to the [Kubectl](../../cluster-management/palette-webctl.md) guide for more information.
+
+### Validate
+
+You can validate your cluster is up and in **Running** state.
 
-The cluster details page of the cluster contains the status and details of the deployment. Use this page to track the deployment progress.
+1. Log in to [Palette](https://console.spectrocloud.com/).
 
+2. Navigate to the left **Main Menu** and select **Clusters**. The **Clusters** page displays a list of all available clusters that Palette manages. 
 
-## Validate
+3. Click on the cluster you created to view its details page.
 
-You can validate your cluster is up and running by reviewing the cluster details page. Navigate to the left **Main Menu** and click on **Clusters**. The **Clusters** page contains a list of all available clusters managed by Palette. Click on the row for the cluster you wish to review its details page. Ensure the **Cluster Status** field contains the value **Running**.
+4. Ensure the **Cluster Status** field displays **Running**.
 
+<br />
 
-## EKS Cluster Secrets Encryption
+## Enable Secrets Encryption for EKS Cluster
 
 Palette encourages using AWS Key Management Service (KMS) to provide envelope encryption of Kubernetes secrets stored in Amazon Elastic Kubernetes Service (EKS) clusters. This encryption is 
-a defense-in-depth security strategy to protect sensitive data such as passwords, docker registry credentials, and TLS keys stored as [Kubernetes Secrets](https://kubernetes.io/docs/concepts/configuration/secret/). 
+a defense-in-depth security strategy to protect sensitive data such as passwords, docker registry credentials, and Transport Layer Security (TLS) keys stored as [Kubernetes Secrets](https://kubernetes.io/docs/concepts/configuration/secret/). 
+
+You can enable secrets encryption when you create an EKS cluster by toggling the **Enable encryption** button and providing the Amazon Resource Name (ARN) of the KMS key. The **Enable encryption** option is available on the cluster creation wizard's **Cluster Config** page for EKS.
 
 ### Prerequisites
 
-* KMS key created in the AWS account.
-* KMS key is of the type symmetric.
-* KMS key policy permits the following actions; encrypt and decrypt.
+- An AWS account added to Palette. Review [Add AWS Account](add-aws-accounts.md) for guidance.
+
+- IAM user or role has attached policies listed in [Required IAM Policies](required-iam-policies.md).
+
+- A **PaletteControllersEKSPolicy** created in AWS and attached to the IAM user or role that Palette is using. To create this policy, refer to [Controllers EKS Policy](required-iam-policies.md#controllers-eks-policy).
+
+

```

Comment:
```suggestion
- A **PaletteControllersEKSPolicy** created in AWS and attached to the IAM user or role that Palette is using. To create this policy, refer to [Controllers EKS Policy](required-iam-policies.md#controllers-eks-policy).


- An AWS KMS key created in the AWS region you intended to deploy cluster to with Palette. 


- Ensure the KMS key policy allows the IAM user or role Palette usage of the KMS key. The KMS key policy must allowed the IAM role or IAM user the following actions:
      - "kms:Encrypt",
      - "kms:Decrypt",
      - "kms:ReEncrypt*",
      - "kms:GenerateDataKey*",
      - "kms:DescribeKey"
  
Check out the [Create a KMS Key policy](https://docs.aws.amazon.com/kms/latest/developerguide/key-policy-overview.html) guide for more information on key policies.     
          
```

Rita, I'm not sure how well my GitHub suggestion will go, so you might have to edit it locally 🙈

Reply:
Incorporated this block locally.